anomaly_detector/anomaly_detector.py

```diff
@@ -70,9 +70,9 @@ def __init__(self, cfg: Configuration):
 
     def _load_data(self, time_span, max_entries, false_positives=None):
         """Loading data from storage into pandas dataframe for processing."""
-        data, raw = self.storage.retrieve(time_range=time_span,
-                                          number_of_entries=max_entries,
-                                          false_data=false_positives)
```

Comment:
should'nt we use named arguments as seen in https://github.com/AICoE/log-anomaly-detector/search?q=retrieve&unscoped_q=retrieve

Reply:
The named arguments is broken. It's failing to work.